blog/2017-09-25-streaming-to-another-database.adoc

```diff
@@ -0,0 +1,222 @@
+= Streaming data to a downstream database
+Jiri Pechanec
+:awestruct-tags: [ mysql, postgres, smt, example ]
+:awestruct-layout: blog-post
+
+Recently there was a short https://www.confluent.io/blog/simplest-useful-kafka-connect-data-pipeline-world-thereabouts-part-1/[series] of blogposts on https://www.confluent.io/[Confluent's] blog.
+
+In this blogpost we will try to create a simple scenario: we are going to have a MySQL database instance running as a master database that feeds a Kafka broker with events representing a source of truth.
+Next we will add PostgreSQL database that will listen one of the topics and will replicate the state of the MySQL instance to its own data.
```

Comment:
"listen *to*"; also, technically, it's not PG itself listening but rather the sink connector.